servers/rendering/renderer_rd/storage_rd/light_storage.cpp

```diff
@@ -2603,6 +2603,26 @@ int LightStorage::get_directional_light_shadow_size(RID p_light_instance) {
 
 LightStorage::ShadowCubemap *LightStorage::_get_shadow_cubemap(int p_size) {
 	if (!shadow_cubemaps.has(p_size)) {
+		// First check if we've already got too many shadow cubemaps allocated
+		// This helps prevent crashes with multiple Omnilight3D lights with cube shadows
+		if (shadow_cubemaps.size() >= MAX_CUBE_SHADOWS) {
+			// Return the existing shadow cubemap with closest matching size
+			int closest_size = 0;
+			int closest_diff = INT_MAX;
+
+			for (const KeyValue<int, ShadowCubemap> &E : shadow_cubemaps) {
+				int diff = ABS(E.key - p_size);
```

Comment:
The `ABS()` macro was removed on March 19 by https://github.com/godotengine/godot/pull/69406 in favor of `Math::abs()`.

I checked out your PR and tried to build it, without success. I'm curious how were you able to compile on your end before submitting the PR, though.